src/plots/gl2d/scene2d.js

```diff
@@ -359,7 +358,6 @@ proto.destroy = function() {
 
     if(!this.staticPlot) this.container.removeChild(this.canvas);
     this.container.removeChild(this.svgContainer);
-    this.container.removeChild(this.mouseContainer);
```

Comment:
The problem is that `mouseContainer` intercepts mousemove, needed for cartesian selection to work. There seems to be no good way to hook up [gl2d/camera](https://github.com/plotly/plotly.js/blob/78b82b85206bde93c7117706278ac698b480139d/src/plots/gl2d/camera.js#L34) so to enable [cartesian/select](https://github.com/plotly/plotly.js/blob/78b82b85206bde93c7117706278ac698b480139d/src/plots/cartesian/select.js#L24) − these are too separate concerns.
It is possible to switch to cartesian hover only for lasso/select mode, but the pick style differs visually, which is confusing.